src/main/java/jenkins/plugins/publish_over_ftp/BapFtpHostConfiguration.java

```diff
@@ -45,8 +43,19 @@
 import org.apache.commons.net.ftp.FTP;
 import org.apache.commons.net.ftp.FTPClient;
 import org.apache.commons.net.ftp.FTPReply;
+import org.apache.commons.net.ftp.FTPSClient;
+import org.apache.commons.net.util.TrustManagerUtils;
 import org.kohsuke.stapler.DataBoundConstructor;
 
+import hudson.Util;
+import hudson.model.Describable;
+import hudson.model.Hudson;
+import hudson.util.Secret;
+import jenkins.plugins.publish_over.BPBuildInfo;
+import jenkins.plugins.publish_over.BPHostConfiguration;
+import jenkins.plugins.publish_over.BapPublisherException;
+import jenkins.plugins.publish_over_ftp.descriptor.BapFtpHostConfigurationDescriptor;
+
```

Comment:
Of course, I preserved this for my first commits, but forgot for the later commits.  I will be more careful in the future.